### PR TITLE
fix: fix the readme file fetcher in create project from template modal

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Fix this bug:
image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-959/fix-template-readme-link

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [x] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/dashboard/templates/tabs/createModal.tsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ export const ProjectTemplateCreateModal = ({ cardTemplate, category }: CreatePro
 
 	const fetchManifestData = async () => {
 		if (!assetDirectory) return;
-		const content = await fetchFileContent(`/assets/templates/${assetDirectory}/README.md`);
-		setManifestData(content);
+		const content = await getTemplateFiles(assetDirectory);
+		setReadme(content["README.md"]);
```
Comment (RonenMars) [inline src/components/organisms/dashboard/templates/tabs/createModal.tsx:60]: Check if we have this key, else set empty string

Comment (J1za) [inline src/components/organisms/dashboard/templates/tabs/createModal.tsx:60]: for example:
```suggestion
		setReadme(content && "README.md" in content ? content["README.md"] : "");
```

Comment (RonenMars) [reply src/components/organisms/dashboard/templates/tabs/createModal.tsx:60]: done

Comment (RonenMars) [reply src/components/organisms/dashboard/templates/tabs/createModal.tsx:60]: refactored

**Pull request merged**